S2VX.Game/Play/PlayScreen.cs

```diff
@@ -12,16 +14,32 @@
 
 namespace S2VX.Game.Play {
     public class PlayScreen : Screen {
-        private S2VXStory Story { get; set; }
+        private string AudioDir { get; set; }
+        private StorageBackedResourceStore CurLevelResourceStore { get; set; }
+        private string CurSelectionPath { get; set; }
+        private string StoryDir { get; set; }
+        private string FullStoryDir { get; set; }
+        public PlayScreen(bool isUsingEditorSettings, string curSelectionPath, string storyDir,
+            StorageBackedResourceStore curLevelResourceStore, string audioDir) {
 
-        public PlayInfoBar PlayInfoBar { get; private set; } = new PlayInfoBar();
+            IsUsingEditorSettings = isUsingEditorSettings;
+            CurSelectionPath = curSelectionPath;
+            StoryDir = storyDir;
+            AudioDir = audioDir;
```

Comment:
Should rename so it's not StoryDir and AudioDir

Reply:
Done